src/main/scala/codacy/brakeman/Brakeman.scala

```diff
@@ -131,6 +131,11 @@ object Brakeman extends Tool {
       case 86 => "ForgerySetting"
       case 87 => "JSONEncoding"
       case 88 => "XMLDoS"
+      case 89 => "SafeBufferManipulation"
+      case 90 => "SendFile"
+      case 91 => "SessionManipulation"
+      case 92 => "WeakHash"
+      case 93 => "CheckModelAttributes"
```

Comment:
This alone does nothing, we have to have the patterns in:
https://github.com/codacy/codacy-brakeman/blob/master/src/main/resources/docs/patterns.json
and
https://github.com/codacy/codacy-brakeman/blob/master/src/main/resources/docs/description/description.json

Check the documentation that I emailed you about our tools integrations for more details.
Or just ask here / email me for anything you need.